### ISSUE TITLE
plot validation loss?

### ISSUE THREAD

**Issue by EvaProg** [unit8co/darts#1429]
Hi! Thank you very much for this extremely helpful library!

My question is, is there a way to access the training and validation losses for each epoch during model fitting? I am currently training an RNN model and would like to plot the training and validation losses to see whether my model is overfitting but I can't seem to find such a functionality. Have I missed something?

**Comment by dennisbader** [comment]
Hey @EvaProg, have a look a this: https://github.com/unit8co/darts/issues/811#issuecomment-1047187100

**Comment by EvaProg** [comment]
Thank you!!
